docs/reference/load-balancer.md

```diff
@@ -0,0 +1,55 @@
+# Load Balancing in Ambassador
+
+Ambassador lets users control how Ambassador load balances between resulting endpoints for a given mapping.
+
+Load balancing configuration can be set for all Ambassador mappings in the [ambassador](https://www.getambassador.io/reference/modules#the-ambassador-module) module, or set per [mapping](https://www.getambassador.io/reference/mappings#configuring-mappings).
+If nothing is set, simple round robin balancing is used via Kubernetes services.
+
+## The `load_balancer` attribute
+
+The `load_balancer` attribute configures the load balancing. The following fields are supported:
+
+- `type`: Specifies the type of load balancer to use.
+- `policy`: Specifies the load balancing policy to use.
+
+### `type: kubernetes`
+The `kubernetes` type delegates load balancing to Kubernetes. Supported policies:
+- `round_robin` (default)
+
```

Comment:
Add: In this mode, Ambassador will route traffic directly to a Kubernetes service. Kubernetes service networking(https://kubernetes.io/docs/concepts/services-networking/) is then responsible for load balancing traffic between each of the pods.